--- /home/hayden/Downloads/dwmblocks.c	2024-04-02 13:46:50.647163189 -0700
+++ dwmblocks.c	2024-04-02 13:44:20.283169630 -0700
@@ -79,7 +79,8 @@
 //opens process *cmd and stores output in *output
 void getcmd(const Block *block, char *output)
 {
-	if (block->signal)
+    // Don't display sig if clickable disabled
+	if (block->signal && CLICKABLE_BLOCKS)
 	{
 		output[0] = block->signal;
 		output++;
@@ -168,11 +169,11 @@
 int getstatus(char *str, char *last)
 {
 	strcpy(last, str);
-	str[0] = '\0';
+    strcpy(str, l_padding);
     for(int i = 0; i < LENGTH(blocks); i++) {
 		strcat(str, statusbar[i]);
         if (i == LENGTH(blocks) - 1)
-            strcat(str, " ");
+            strcat(str, r_padding);
     }
 	str[strlen(str)-1] = '\0';
 	return strcmp(str, last);//0 if they are the same
